wiki/FAQ/en.md

```diff
@@ -56,161 +78,225 @@ _[Osu! Tatakae! Ouendan][ouendan wikipedia]_ is a rhythm game for the Nintendo D
+
+This is a spinner.
+
+You will have to click and hold the hit-button. From there, use the mouse (or your finger/pen) and spin the spinner in a circular motion (in either direction) until the spinner circle grows outwards completely. A _Clear_ notice will appear to indicate that the spinner was completed. If the spinner was cleared early, continue spinning to collect score bonus and gain some health back.
+
+The outer white circle shows how much time left to complete the spinner. This circle will turn red to notify that time is almost out. Older beatmaps, using skin version 1.0, will have a meter/gauge to indicate how close to clearing the spinner.
+
+The small box below the spinner shows your spins per minute. The number inside that box translates to the number of spins you could make, if you were to continue to spin the spinner at that speed for one minute. Know that you will be unable to spin the spinner, if you did not hold any of the hit-buttons prior to spinning.
+
+On the DS, this is a good way to scratch your touchscreen (or screen protectors), especially on the higher difficulties. In osu!, spinners have been toned down as to not be such a nuisance to complete. The maximum number of spins per minute achievable is 477 which is also the rate at which the [Auto][Auto wikilink] game modifier spins while the [Spun Out][Spun Out wikilink] game modifier spins slower at 286 spins per minute.
 
 #### In-Game Interface
 ![osu! interface][osu! Interface image]
 
-This is the osu! game style interface.
--   The top left bar is your "health".
-  -   This will decrease at a steady rate \[depending on the beatmap difficulty set by the mapper(s)\], but you can replenish it by hitting notes at the right time.
-  -   A perfectly timed hit (300 hit or Geki Beat) will increase your health more than a badly timed hit (50 hit).
-  -   A total miss will take a good chunk out of your health.
--   Next to the health bar (at the right) is your score.
--   Below the score is your accuracy.
--   The circle beside your accuracy and below your score is a timer bar, which shows the song length.
--   The number on the bottom left is the combo counter/score multiplier.
+This is the in-game interface for osu!.
+
+The top left bar is the healthbar. This will decrease at a steady rate (depending on the beatmap difficulty set by the mapper), but can be replenished by tapping notes at the right time or spinning the spinner. A perfectly timed hit (a 300 or Geki) will increase your health more than a badly timed hit (50). A total miss will take a good chunk of your health out.
+
+On the right of the health bar is the score. Below that is the accuracy. The circle beside the accuracy (and below the score) is a timer for the duration of the beatmap. The number on the bottom left is the combo counter/score multiplier.
 
 ---
 
 ## Alternative Game Modes
+### How to change game mode?
```

Comment:
Change header to
`How do I change the game mode?`

Or 

`How to change the game mode`
Should be fine, I have guess